src/core/server/metrics/metrics_service.ts

```diff
@@ -84,4 +93,17 @@ export class MetricsService
     }
     this.metrics$.complete();
   }
+
+  private setupOpsMetricsEventType(
+    analytics: AnalyticsServiceSetup,
+    metricsObservable: Observable<OpsMetrics>
+  ) {
+    analytics.registerEventType<OpsMetrics>({
+      eventType: 'core-ops_metrics',
+      schema: opsMetricsSchema,
+    });
+    metricsObservable.subscribe((metrics: OpsMetrics) => {
+      analytics.reportEvent('core-ops_metrics', { ...metrics });
```

Comment:
in adding the functional tests for reporting ops metrics events, I realized we were sending a summary of the ops metrics, as they are logged out. We don't have validation between the event schema and event we're reporting right now so that didn't catch it.

We're more likely to be interested in the detailed `opsMetrics` over time, although the documents will be somewhat larger than the condensed log format although I don't suspect by much.

What that got me thinking though is _should we consider sending events in ECS-compatible format_?

Reply:
Not sure about ECS format (@afharo will probably have a better idea on that), but my gut feeling is that we want want a layer of conversion between the raw metrics and the event data.

using `analytics.reportEvent('core-ops_metrics', { ...metrics })` directly looks like bad isolation ihmo. 

I would expect something like  `analytics.reportEvent('core-ops_metrics', convertToMetricEvent(metrics) )` (even if the function just returns `{ ...metrics }` for now

Reply:
> my gut feeling is that we want a layer of conversion between the raw metrics and the event data.

I agree! We might want to limit some of the data that we send. There are 2 ongoing discussions related to that:

1. `process` vs. `processes`: `process` is just `processes[0]`, and it's deprecated, so probably a good candidate to be removed. https://github.com/elastic/kibana/pull/132298#discussion_r875793988
2. `control_group` might not be PII safe? https://github.com/elastic/kibana/pull/132298#discussion_r876810901

Reply:
> There are 2 ongoing discussions related to that

1. is done, we're not sending duplicate data now
2. From my research, `control_group` is PII safe, it has nothing to do with auth.